Tools/CISupport/ews-build/steps.py

```diff
@@ -3510,6 +3511,10 @@ def send_email_for_flaky_failure(self, test_name):
 class RunWebKitTestsWithoutChange(RunWebKitTests):
     name = 'run-layout-tests-without-change'
 
+    def start(self):
+        self.workerEnvironment[RESULTS_SERVER_API_KEY] = os.getenv(RESULTS_SERVER_API_KEY)
```

Comment:
Is it worth logging a warning here if we aren't able to get `RESULTS_SERVER_API_KEY`? Seems like there's a potential to fail silently otherwise.